src/nat/cli/commands/info/info.py

```
@@ -34,4 +33,19 @@ def info_command(**kwargs):
+
+
+@click.command(
+    name="mcp",
+    help="Removed. Use 'nat mcp client' instead.",
+)
+def info_mcp_deprecated():
+    """
+    Removing support for the old 'nat info mcp' command.
+    """
+    raise click.UsageError("The 'nat info mcp' command has been removed. "
+                           "Use the new 'nat mcp client' commands instead")
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)